src/Instrumentation/Slim/src/CallableFormatter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Instrumentation\Slim;
+
+use Closure;
+use ReflectionFunction;
+
+class CallableFormatter
+{
+    /**
+     * @see https://stackoverflow.com/a/68113840/2063413
+     * @phan-suppress PhanUndeclaredMethod
+     * @psalm-suppress UndefinedMethod
+     */
+    public static function format($callable): string
+    {
+        if ($callable instanceof Closure || (is_string($callable) && function_exists($callable))) {
+            return (new ReflectionFunction($callable))->getName();
```

Comment:
Has to handle `Clazz::method(...)` and `$object->method(...)` too.

Reply:
I _think_ this is implemented now for both static and non-static class methods

Reply:
Done. It isn't pretty, but output now matches suggestions above.